### PR TITLE
task 13

### PR DESCRIPTION
**Pull request by nquocviet**
https://edu-redmine.sun-asterisk.vn/issues/34838

### CODE REVIEW

**Comment by nquocviet** [comment]
ready

File: dist/index.html [hunk 1]
```diff
@@ -1947,24 +1947,29 @@ <h1 class="title">Sản phẩm bán chạy</h1>
       <div class="container">
         <div class="row">
           <div class="col-lg-8 col-md-6">
+<<<<<<< HEAD
             <div class="news-section">
               <h1 class="title">Tin tức &amp; blog</h1>
+=======
+            <div class="blog-section">
+              <h1 class="title">Tin tức & blog</h1>
+>>>>>>> c2cbddd (task 13)
```
Comment (manhnt-0960) [inline dist/index.html]: .

File: pages/index.pug [hunk 2]
```diff
@@ -92,29 +94,18 @@ block layout
         div.container
             div.row
                 div.col-lg-8.col-md-6
+<<<<<<< HEAD
                     div.news-section
                         h1.title Tin tức &amp; blog
+=======
+                    div.blog-section
+                        h1.title Tin tức & blog
+>>>>>>> c2cbddd (task 13)
```
Comment (manhnt-0960) [inline pages/index.pug]: .

**Comment by nquocviet** [comment]
ready

**Pull request merged**